sandbox_windows.go

```diff
@@ -0,0 +1,12 @@
+// +build windows
+
+package libnetwork
+
+import "github.com/docker/libnetwork/osl"
+
+func (sb *sandbox) Key() string {
+	if sb.config.useDefaultSandBox {
+		return osl.GenerateKey("default")
+	}
+	return osl.GenerateKey(sb.containerID)
```

Comment:
Thinking out loud here; Instead of making an exception here for windows, would it make sense to set `sb.id` to containerID when creating the sandbox? https://github.com/docker/libnetwork/blob/b5cc5c5dee7345ae2356540f235af94ee67af7e9/controller.go#L1018

Reply:
Am looking at this option. Am not sure if that would break any other assumption

Reply:
Not sure either, but was assuming that "id" would be treated as an opaque value; and if that works, no logic has to be changed down the line.

@mavenugo will be more familiar with that, and may be able to add some input on that

Reply:
@thaJeztah am going with your suggestion which makes the changes simpler.
Am running a bunch of tests to make sure that it doesn't break anything on windows.